packages/algob/src/lib/algo-operator.ts

```diff
@@ -226,6 +236,89 @@ export class AlgoOperatorImpl implements AlgoOperator {
     };
   }
 
+  /**
+   * Description: Function to deploy Stateful Smart Contract
+   * @param approvalProgram name of file in which approval program is stored
+   * @param clearProgram name of file in which clear program is stored
+   * @param flags         SSCDeploymentFlags
+   * @param payFlags      TxParams
+   * @param txWriter
+   */
+  async deploySSC (
+    approvalProgram: string,
+    clearProgram: string,
+    flags: SSCDeploymentFlags,
+    payFlags: TxParams,
+    txWriter: txWriter): Promise<SSCInfo> {
+    const sender = flags.sender.addr;
+    const params = await tx.mkSuggestedParams(this.algodClient, payFlags);
+
+    const onComplete = algosdk.OnApplicationComplete.NoOpOC;
+
+    const app = (await this.ensureCompiled(approvalProgram, false));
+    const approvalProg = new Uint8Array(Buffer.from(app.compiled, "base64"));
+    const clear = (await this.ensureCompiled(clearProgram, false));
+    const clearProg = new Uint8Array(Buffer.from(clear.compiled, "base64"));
+
+    let message = "Signed transaction with txID: ";
+    const txn = algosdk.makeApplicationCreateTxn(
+      sender,
+      params,
+      onComplete,
+      approvalProg,
+      clearProg,
+      flags.localInts,
+      flags.localBytes,
+      flags.globalInts,
+      flags.globalBytes,
+      flags.appArgs,
+      flags.accounts,
+      flags.foreignApps,
+      flags.foreignAssets,
+      flags.note,
+      flags.lease,
+      flags.rekeyTo);
+
+    const txId = txn.txID().toString();
+    const signedTxn = txn.signTxn(flags.sender.sk);
+    message = message.concat(txId);
+    message = message.concat("\nCreated new app-id: ");
+
+    const txInfo = await this.algodClient.sendRawTransaction(signedTxn).do();
+
+    const confirmedTxInfo = await this.waitForConfirmation(txId);
+
+    const appId = confirmedTxInfo['application-index'];
+    message = message.concat(appId.toString());
+
+    console.log(message);
```

Comment:
why concatenating message, instead of doing a simple log here:
```
console.log("Signed transaction with txID: %s\nCreated new app-id: %s", txId, appId)
```

Reply:
did this because, this message is also logged in transaction logs with TxInfo

Reply:
Then let's use string formatting:
```
message = "Signed transaction with txID: ${txId}\n ...."
```

Reply:
ok